db/fixtures/lessons/git_lessons.rb

```diff
@@ -35,5 +35,14 @@ def git_lessons
       github_path: '/git/intermediate_git/using_git_in_the_real_world.md',
       identifier_uuid: 'c8b7ccc1-8a16-4545-9d46-b9091d45c6b4',
     },
+
+    {
+  'Working with Remotes' => {
```

Comment:
Nit: Indentation is off for this new lesson, you can use the other lessons in this file as a reference. The 
indentation used should match and be on the same level as the others.